### ISSUE TITLE
Was the DwC Maintenance Group charter approved?

### ISSUE THREAD

**Issue by baskaufs** [tdwg/dwc#163]
Does anybody know if the exec approved the charter for the maintenance interest group?  If so, we should put the charter somewhere in the repo and list the core members somewhere easily found.

**Comment by peterdesmet** [comment]
Dimitris just announced it: it was approved! 🎉  Charter should be on the (new) website, not sure it should be here as well... but could. I don't like to maintain it in 2 places.

**Comment by baskaufs** [comment]
I don't think the charter itself is as important as informing people about who the key people are in the IG.  That is, they should know who to contact if they have questions or issues.  If that list will be maintained elsewhere, but people will be landing here, then we could have a link from the home readme that directs them to the list.

**Comment by gkampmeier** [comment]
I think the charter is required to be a public document associated with the group. But @baskaufs is right that ultimately users will find key people most useful.

**Comment by peterdesmet** [comment]
1. The charter will go on the website
2. The README should eventually also link to the page on the website
3. There is a link to the contributors of this repo [in the README](https://github.com/tdwg/dwc#contributors), which - although not necessarily the same as the core members of the maintenance group - I always find useful, as you see who's actually involved. 
4. The README also has a link to the dwc-qa: https://github.com/tdwg/dwc#getting-started
5. We have a @dwc team on GitHub: I've just updated its name to Darwin Core maintenance group  and verified that the group members are the same as on the charter

I hope that is sufficient for people to find their way around and knowing who's involved and who to contact. Closing issue.
